File: src/main/java/me/mapetr/simpleutilities/commands/Kill.java
package me.mapetr.simpleutilities.commands;

import co.aikar.commands.BaseCommand;
import co.aikar.commands.annotation.CommandAlias;
import co.aikar.commands.annotation.Default;
import co.aikar.commands.annotation.Description;
import org.bukkit.entity.Player;

@CommandAlias("kys|die|suicide")
public class Kill extends BaseCommand {
    @Default
    @Description("Kills you")
    public void onCommand(Player player) {
        player.setHealth(0);
    }
}

File: src/main/java/me/mapetr/uwuEssentials/commands/Kill.kt
package me.mapetr.uwuEssentials.commands

import co.aikar.commands.BaseCommand
import co.aikar.commands.annotation.CommandAlias
import co.aikar.commands.annotation.Default
import co.aikar.commands.annotation.Description
import org.bukkit.entity.Player

@CommandAlias("kys|die|suicide")
class Kill : BaseCommand() {
    @Default
    @Description("Kills you")
    fun onCommand(player: Player) {
        player.health = 0.0
    }
}
